debian/changelog

```diff
@@ -1,3 +1,21 @@
+felix (2.6.0~__STREAM__) __STREAM__; urgency=low
+
+  * Felix 2.6.0 (from Git commit 64caa62).
+    [Changes recorded in 2.6.0 tag]
+    By upgrading libcalico-go to v1.7.0, we add support for Kubernetes'
+    v1.8.0 NetworkPolicy with Egress rule and IPBlock functionality.
+    
+    In addition, libcalico-go now supports a new `Types` field which
+    specifies whether a rule should apply to `ingress`, `egress` or
+    both types of traffic.
+    
+    - Allow Policy to explicitly govern ingress and/or egress [#1557](https://github.com/projectcalico/felix/pull/1557)
+    - Add read/write timeout options for Typha connection. [#1538](https://github.com/projectcalico/felix/pull/1538)
+    - Fix OpenStack detection heuristic to ignore 'none'. [#1556](https://github.com/projectcalico/felix/pull/1556)
+    - Adding support for ppc64le. [#1516](https://github.com/projectcalico/felix/pull/1516)
+
+ -- Neil Jerram <neil@tigera.io>  Tue, 26 Sep 2017 13:06:57 -0700
```

Comment:
This should really be your name and email.  (I'm not sure that enough time has elapsed since -rc2 for me to travel between a +0100 and a -0700 time zone :smile: )